run-test.sh

```diff
@@ -48,7 +48,7 @@ usage()
     echo "    --restrict-proj <regex>       Run test projects that match regex"
```

Comment:
We've been talking about deprecating this script for a while. In the meantime, a couple of syntax errors were causing the script to *defintely* fail :) This was purely a quick syntax fix to see if testing in CI would cooperate.